common/include/assign_port.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include "verilated.h"
+#include "verilog_int.h"
+
```

Comment:
I think name like write_verilator_port is better, because you will need somethine like read_verilator_port.
My suggestion: something like verilog::write_port()?

Reply:
Use write_verilator_port.